toolbox/api/datagalaxy_api_dictionary.py

```diff
@@ -0,0 +1,161 @@
+import logging
+
+import requests as requests
+from toolbox.api.datagalaxy_api_workspaces import DataGalaxyBulkResult, to_bulk_properties_tree
+
+
+class DataGalaxyApiDictionary:
+    def __init__(self, url: str, access_token: str, workspace: dict):
+        self.url = url
+        self.access_token = access_token
+        self.workspace = workspace
+
+    def list_sources(self, workspace_name: str) -> list:
+        if not self.workspace["isVersioningEnabled"]:
```

Comment:
I find it easier to read if the raise is at the top
```suggestion
        if self.workspace["isVersioningEnabled"]:
            raise Exception('Workspace with versioning enabled are currently not supported.')
```

In this case it removes the negation which is nice.